LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014 Peter Berkenbosch and contributors.
+Copyright (c) 2020 [name of plugin creator]
```

Comment:
We should find a way to fix this in `solidus_dev_support`, maybe not touching the LICENSE if it already exists…